driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -223,9 +222,27 @@ impl PriceFinding for OptimisationPriceFinder {
             orders: orders.iter().map(From::from).collect(),
             fee: self.fee.as_ref().map(From::from),
         };
-        let current_time = Utc::now().to_rfc3339();
-        let input_file = format!("instances/instance_{}.json", &current_time);
-        let result_folder = format!("results/instance_{}/", &current_time);
+
+        let now = Utc::now();
+        let batch_id = now.timestamp() / 300;
```

Comment:
Should it be `now.timestamp() / 300 - 1`?

Reply:
Makes sense to me as we are solving for the previous batch.

Reply:
Great catch!

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia0.giphy.com%2Fmedia%2F3o6ZsSRkXifoEJfpMA%2Fgiphy-downsized-medium.gif&token=great_catch" data-gitmeme-token="great_catch"><img src="https://media0.giphy.com/media/3o6ZsSRkXifoEJfpMA/giphy-downsized-medium.gif" title="Created by gitme.me with /great_catch"/></a>